inspirehep/submissions/views.py

```diff
@@ -156,6 +165,164 @@ def start_workflow_for_submission(self, submission_data, control_number=None):
         abort(503)
 
 
+class JobSubmissionsResource(BaseSubmissionsResource):
+    decorators = [login_required_with_roles()]
+    user_allowed_status_changes = {
+        "pending": ["pending"],
+        "open": ["open", "closed"],
+        "closed": ["closed"],
+    }
+
+    def get(self, pid_value):
+        try:
+            pid, _ = pid_value.data
+            record = JobsRecord.get_record_by_pid_value(pid.pid_value)
+        except PIDDoesNotExistError:
+            abort(404)
+
+        serialized_record = job_v1.dump(record)
+        return jsonify({"data": serialized_record})
+
+    def post(self):
+        """Adds new job record"""
+        data = job_loader_v1()
+        data = self.prepare_data(data)
+        record = JobsRecord.create(data)
+        db.session.commit()
+        self.create_ticket(record, "rt/new_job.html")
+        response = jsonify({"status": "created", "pid_value": record["control_number"]})
+        response.status_code = 201
+
+        return response
```

Comment:
you can just return ``jsonify({}), 201``, IMHO we don' need the status, we can check the status_code if we need

Reply:
I need pid_value for tests. I told you that. It won't hurt anyone, and it helps testing stuff.
And to be consistent with submit author endpoint where we return workflow_id, this pid_value is required.

Reply:
I'm not talking about the ``pid_value``, but how to specify the status code :)

``jsonify({........}), 201``

Reply:
o, ok

Reply:
Still I would prefer to stay with setting status code explicitly. It's more readable.

Reply:
this is the flask way